### DISCUSSION TITLE
New SBOM Export is lacking many SPDX fields

### DISCUSSION THREAD

**Discussion by TheModemLisa** [community/community#52168] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

In reviewing SBOM exports across multiple repositories in different languages I have not found any that include more than 8 of the 27 SPDX specification fields. I also could not locate any documentation on what to expect in the export, how to troubleshoot issues such as "NOASSERTION" results or if the responsibility of the accuracy of the export is up to the repository owner or GitHub. As collection of downstream SBOMs from public repositories will become common practice with the engagement of the [Executive Order 14028](https://www.nist.gov/itl/executive-order-14028-improving-nations-cybersecurity), these questions should be critical to many organizations.
I've read through the available documentation and I do not see any instructions for modification of the fields available in the export and the link in the documentation takes you directly to the SPDX specification noting all 27 fields.

**Comment by Dwarfex** [comment]
Yap - here too. Especially missing License information here - in the Web GUI the Packages are correctly referenced and have a lot more info (which is missing in the SBOM)
